modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/db/dao/EpisodeDao.kt

```diff
@@ -46,7 +46,7 @@ abstract class EpisodeDao {
     @Query("SELECT * FROM podcast_episodes WHERE uuid IN (:episodeUuids)")
     abstract suspend fun findByUuids(episodeUuids: List<String>): List<PodcastEpisode>
 
-    @Query("SELECT * FROM podcast_episodes WHERE podcast_id = :podcastUuid AND played_up_to != 0.0")
+    @Query("SELECT * FROM podcast_episodes WHERE podcast_id = :podcastUuid AND played_up_to > (duration / 2)")
     abstract suspend fun findPlayedEpisodesFrom(podcastUuid: String): List<PodcastEpisode>
```

Comment:
As we don't need the episode data this could be a count, but I don't mind either way. 
```suggestion
    @Query("SELECT count(*) FROM podcast_episodes WHERE podcast_id = :podcastUuid AND played_up_to > (duration / 2)")
    abstract suspend fun countPlayedEpisodes(podcastUuid: String): Int
```